### PR TITLE
docs: Add table outlining Issue Tags

### PR DESCRIPTION
**Pull request by furi0us333**
This KB article will be helpful for people writing custom policies that want to include specific issue types.

### CODE REVIEW

File: docs/knowledge_base/issue_tags.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  - Malicious Code (`M`)
+  - Vulnerabilities (`V`)
+  - License Risk (`L`)
+    
+
+| Tag ID | Issue Name | Issue Description |
+| --- | --- | --- |
+| CA0001    | [Bad Author](https://docs.phylum.io/docs/bad_author) | Author is known malicious |
+| CM0001    | [IP Detection](https://docs.phylum.io/docs/ip_identification) | Package contains suspicious IP addresses |
+| CM0003    | Landing Binary | Package is known to contain malware |
```
Comment (louislang) [inline docs/knowledge_base/issue_tags.md:29]: This isn't correct. Landing binary is just that the package uses LOL binaries. Maybe update to say "Package is using LOL binaries in a malicious way"?

**Review by louislang — CHANGES_REQUESTED** [review]
Looking over the `rules` repo, the `Basic Javascript Obfuscation` is indicated as in Prod, but not enabled. This is wrong, it's enabled in prod. We should add it here too.

File: docs/knowledge_base/issue_tags.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+| HM0025    | [Environment Variable Enumeration](https://docs.phylum.io/docs/env_var_enumeration) | Package enumerates through user folders |
+| IE0027    | [Trivial Package](https://docs.phylum.io/docs/trivial_package) | Package may be too small to be worth the security risk |
+| MM0028    | [Suspicious URL References](https://docs.phylum.io/docs/suspicious_url_references) | Package references pastebin-like sites |
+| HM0029    | [Obfuscated Python](https://docs.phylum.io/docs/obfuscated_python) | Package contains obfuscated Python |
+| .M0031    | [Suspicious Commands In Setup.py](https://docs.phylum.io/docs/suspicious_setup_commands) | Package contains unusual commands in setup.py |
+| HM0032    | [Exec on Remote URL](https://docs.phylum.io/docs/executes_code_at_remote_url) | Package calls `exec` on a remote URL |
+| HM0036    | [Webhook Exfil](https://docs.phylum.io/docs/webhook_exfil) | A package that appears to exfil data through a webhook |
+| HM0037    | Malware Bazaar Check | A package's file contains a hash that hits on Malware Bazaar |
+| CM0038    | Triaged Malware Rule (via threat feed) | Manually reviewed and confirmed to contain malware |
```
Comment (louislang) [inline docs/knowledge_base/issue_tags.md:57]:
```suggestion
| CM0038 | Triaged Malware Rule (via threat feed) | Manually reviewed and confirmed to contain malware |
| HM0099 | Obfuscated Javascript | Package contains obfuscated Javascript |
```

File: docs/knowledge_base/issue_tags.md (hunk 1 above)
Comment (spellman) [reply docs/knowledge_base/issue_tags.md:29]: @louislang I got that straight from the issue text: https://github.com/phylum-dev/rules/blob/06fe2e10e8726ac2b663aef3b59cf0688e4b180e/src/rules/landing_binary_rule.py#L20

Shall I update the issue text to say, "Package is using LOL binaries in a malicious way"?

File: docs/knowledge_base/issue_tags.md (hunk 2 above)
Comment (spellman) [reply docs/knowledge_base/issue_tags.md:57]: I had this correct in my spreadsheet and wrote it incorrectly in the table. https://docs.google.com/spreadsheets/d/12bCbis85FmSSw4Nm9wYQtC3dEX8f-45xGj-xny45-eA/edit?usp=sharing

I'll check those one more time and make a PR to update the table in the rules repo.

Comment (louislang) [reply docs/knowledge_base/issue_tags.md:57]: Thanks @spellman!

File: docs/knowledge_base/issue_tags.md (hunk 1 above)
Comment (louislang) [reply docs/knowledge_base/issue_tags.md:29]: oh, I think I see. That's some old stuff from super _super_ early Phylum days: https://github.com/phylum-dev/rules/blob/06fe2e10e8726ac2b663aef3b59cf0688e4b180e/src/rules/landing_binary_rule.py#L14

I think this is overcome by the curated malware stuff. I'd leave it and update the text like you said, but in practice idk that anyone will run into it.

Comment (furi0us333) [reply docs/knowledge_base/issue_tags.md:29]: Final answer - how do you want this updated?

Comment (louislang) [reply docs/knowledge_base/issue_tags.md:29]: "Package is using living off the land binaries in a known malicious way"

**Review by louislang — APPROVED** [review]

**Review by spellman — APPROVED** [review]

**Pull request merged**